src/components/Primary-Button/Button.constants.ts

```diff
@@ -0,0 +1,58 @@
+import type { ButtonVariant, ButtonSize, ButtonShape } from './Button.types';
+
+export const BUTTON_VARIANTS = {
+  primary: 'primary',
+  secondary: 'secondary',
+} as const;
+
+export const BUTTON_SIZES = {
+  small: 'small',
+  medium: 'medium',
+  large: 'large',
+} as const;
+
+export const BUTTON_SHAPES = {
+  rounded: 'rounded',
+  square: 'square',
+  pill: 'pill',
+} as const;
+
+export const SIZE_CONFIGS = {
```

Comment:
SIZE_CONFIGS contains pixel values (height, minWidth) and rem values. Component-level constants should avoid fixed pixel sizes where possible; prefer design tokens or spacing units derived from theme so sizing scales consistently across themes and devices.

Suggestion:
- Define sizes in token terms (e.g. spacing multiples) or move the final calculation to the styles file where theme is available. For example, store paddingScale and a heightMultiplier here, then compute actual px via theme.spacing in Button.styles.